machine_management/user_infra/adding-compute-user-infra-general.adoc

```diff
@@ -0,0 +1,32 @@
+[id="adding-compute-user-infra-general"]
+= Adding compute machines to clusters with user-provisioned infrastructure
+include::modules/common-attributes.adoc[]
+:context: adding-compute-user-infra-general
+
+toc::[]
+
+In addition to adding compute machines to a cluster during the user-provisioned infrastructure installation process, you can add more compute machines to a cluster with user-provisioned infrastructure. The process requires some of the same configuration files and parameters used during installation.
```

Comment:
The first sentence confused me a bit. What about something like this:

`You can add compute machines to a cluster on user-provisioned infrastructure either as part of the installation process or after installation. The post-installation process, shown here, requires some of the same configuration files and parameters that were used during installation.`

Is that clearer?